huaweicloud/services/lb/resource_huaweicloud_lb_loadbalancer.go

```diff
@@ -126,16 +126,16 @@ func ResourceLoadBalancerV2() *schema.Resource {
 }
 
 func resourceLoadBalancerV2Create(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	config := meta.(*config.Config)
-	elbClient, err := config.LoadBalancerClient(config.GetRegion(d))
+	cfg := meta.(*config.Config)
+	elbClient, err := cfg.LoadBalancerClient(cfg.GetRegion(d))
```

Comment:
we can use a variable to save the region, just like: ```region := cfg.GetRegion(d)```

Reply:
done